hips/hip-0012.md

```diff
@@ -44,153 +144,117 @@ The person will have the following responsibilities:
 - Determine the timelines for development
 - Make the determination of when the project has indeed reached a milestone
 - Name and oversee releases
-- After the release, the release manager will determine what the "intent" of a Helm behavior was (which informs determining whether an incoming breaking change is a feature or a bug fix)
+- After the release, the release engineer will determine what the "intent" of a Helm behavior was (which informs determining whether an incoming breaking change is a feature or a bug fix)
 
 #### Determining Who Will Be Release Engineer
 
-The release engineer must be a core maintainer on the `github.com/helm/helm` project.
-The engineer must not be in danger of losing core maintainer-ship due to inactivity.
-Anyone can nominate a core maintainer to become the release engineer (self-nominations are encouraged).
-Any nominee must _agree_ to the nomination.
-The Helm project maintainers will determine which of the nominated maintainers shall be the release engineer.
+To become the release engineer for the Helm project, an individual must be a core maintainer on <https://github.com/helm/helm> project.
+The engineer should not be at risk of losing their core maintainer status due to inactivity.
```

Comment:
This is the original verbiage. I think it means we can't choose a release engineer who isn't an active core maintainer.

It can change to just saw that: release engineer must be an active maintainer